pkg/util/system/system_utils_test.go

```diff
@@ -21,27 +21,89 @@ import (
 
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 )
 
 func TestIsMasterNode(t *testing.T) {
+	labelMater := map[string]string{
+		constants.LabelNodeRoleMaster: string(v1.TaintEffectNoSchedule),
+	}
 	testCases := []struct {
-		input  string
+		node   v1.Node
 		result bool
 	}{
-		{"foo-master", true},
-		{"foo-master-", false},
-		{"foo-master-a", false},
-		{"foo-master-ab", false},
-		{"foo-master-abc", true},
```

Comment:
@xiangpengzhao The nodeName[:len(nodeName)-3] will be 'foo-master-', strings.HasSuffix() just check the string which end with 'master-', so its result is true.